manifests/j/JFrog/Conan/1.39.0/JFrog.Conan.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JFrog.Conan
+PackageVersion: 1.39.0
+PackageLocale: en-US
+Publisher: JFrog
+PublisherUrl: https://conan.io/
+PrivacyUrl: https://conan.io/privacy-policy.html
+PackageName: Conan
+PackageUrl: https://conan.io/
+License: MIT
+LicenseUrl: https://raw.githubusercontent.com/conan-io/conan/develop/LICENSE.md

```

* Changed PackageName to "Conan Package Manager"
* Added PublisherSupportUrl
* Added Copyright and Copyright Url
```suggestion
Publisher: JFrog
PublisherUrl: https://conan.io
PublisherSupportUrl: https://github.com/conan-io/conan/issues
PrivacyUrl: https://conan.io/privacy-policy.html
PackageName: Conan Package Manager
PackageUrl: https://conan.io
License: MIT
LicenseUrl: https://raw.githubusercontent.com/conan-io/conan/develop/LICENSE.md
Copyright: Copyright (c) 2019 JFrog LTD
CopyrightUrl: https://raw.githubusercontent.com/conan-io/conan/develop/LICENSE.md
```